packages/app/src/overview/components/update-os-modals/update-available-modal/about-updates-section.component.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { defineMessages } from "react-intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+import { borderColor } from "App/__deprecated__/renderer/styles/theming/theme-getters"
+
+const messages = defineMessages({
+  updateAvailableAboutUpdatesTitle: {
+    id: "module.overview.updateAvailableAboutUpdatesTitle",
+  },
+  updateAvailableAboutUpdatesOsVersionDescription: {
+    id: "module.overview.updateAvailableAboutUpdatesOsVersionDescription",
+  },
+  updateAvailableAboutUpdatesTargetOsVersionSubDescription: {
+    id: "module.overview.updateAvailableAboutUpdatesTargetOsVersionSubDescription",
+  },
+})
+interface Release {
+  version: string
+  date: Date
+}
+
+export interface AboutUpdatesSectionProps {
+  releases: Release[]
+}
+
+const AboutUpdatesSectionTitle = styled(Text)`
+  border-bottom: 0.1rem solid ${borderColor("secondary")};
+  margin-bottom: 1rem;
+`
+
+const AboutUpdatesSectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 2.4rem 0;
+
+  p {
+    text-align: left;
+  }
+`
+
+const UpdatesOsVersionContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+`
```

Comment:
Maybe those should be exported to separated files (`.styled.tsx` and `.interface.ts`)?

Reply:
Not a big problem for me, I just think that the approach follows our project pattern + (at least for me) it increases readability (the file is less overcrowded) + it avoid adding (not so heavy, but annoying) work for future in case this component would require more complicated interface or more styled elements.

Not forcing though, as you said, for this moment it is a small component and for me this version is also ok :)

Reply:
I think that the optional (for a small component) was the one of the our project pattern 😉